.github/workflows/sdk-test.yaml

```diff
@@ -0,0 +1,60 @@
+name: Speakeasy Test SDK
+
+on:
+  workflow_call:
+    inputs:
+      target:
+        description: "The specific target to test"
+        required: false
+        type: string
+      working_directory:
+        description: "The working directory for running Speakeasy CLI commands in the action"
+        required: false
+        type: string
+    secrets:
+      github_access_token:
+        description: A GitHub access token with read access to the repo
+        required: true
+      speakeasy_api_key:
+        description: The API key to use to authenticate the Speakeasy CLI
+        required: true
+jobs:
+  test:
+    name: Test SDK
+    runs-on: ubuntu-latest
+    steps:
+      - name: Tune GitHub-hosted runner network
+        uses: smorimoto/tune-github-hosted-runner-network@v1
+
+      - name: Check commit message condition
+        id: check_commit
+        run: |
+          if [[ "${{ github.event_name }}" != "push" ]]; then
+            echo "Skipping commit message check since event is not push."
+            echo "run_tests=true" >> $GITHUB_OUTPUT
```

Comment:
Should this be false?

Reply:
No this is a very specific check for PUSH events

Reply:
If it's a PUSH event we want to avoid duplicate runs. We only have this to support our somewhat hacky app based empty commit which will have that special message